code/game/turfs/open/floor/plating/wasteplanet.dm

```diff
@@ -48,4 +53,45 @@
 /turf/open/water/tar/waste/lit
 	light_range = 2
 	light_power = 0.2
+	light_color = LIGHT_COLOR_LAVA
+
+/turf/open/water/waste
+	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/water/waste/lit
```

Comment:
define the lits where the rest of them are defined in file. That is to say at the bottom.